### ISSUE TITLE
[Bug]: nvidia-smi unhandled exception: invalid literal for int() with base 10: 'N/A'

### ISSUE THREAD

**Issue by koebi001** [netdata/netdata#15229]
### Bug description

On my system with old Nivida Tesla K40c cards the `nvidia_smi` plugin does no work because `tx_util` and `rx_util` have the value `N/A`.

### Expected behavior

No ERROR.

### Steps to reproduce

1. on a system with a Nvidia Tesla K40c installed enabel the pyton modul `nvidia_smi`
    set `nvidia_smi: yes` in file /etc/netdata/python.d.conf
2. restart netdata
...

### Installation method

kickstart.sh

### System info

```shell
Linux aim-gpu3 6.1.0-9-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.27-1 (2023-05-08) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="12"
/etc/os-release:VERSION="12 (bookworm)"
/etc/os-release:VERSION_CODENAME=bookworm
/etc/os-release:ID=debian
```

### Netdata build info

```shell
Version: netdata v1.40.0
Configure options:  '--build=x86_64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-option-checking' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' '--disable-maintainer-mode' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' '--disable-dependency-tracking' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-z,relro' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: x86_64
    Packaging distro:  
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
    HTTPD (h2o):                YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    debugfs:                 YES
    EBPF:                    YES
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

### Additional info

```
$ ./python.d.plugin trace debug nolock nvidia_smi
2023-06-21 15:06:36: python.d DEBUG: nvidia_smi[nvidia_smi] : started, update frequency: 1
2023-06-21 15:06:37: python.d DEBUG: nvidia_smi[nvidia_smi] : get_data() returned no data
2023-06-21 15:06:37: python.d DEBUG: nvidia_smi[nvidia_smi] : update => [FAILED] (elapsed time: -, failed retries in a row: 1)
2023-06-21 15:06:38: python.d ERROR: nvidia_smi[nvidia_smi] : update() unhandled exception: invalid literal for int() with base 10: 'N/A'
2023-06-21 15:06:38: python.d ERROR: nvidia_smi[nvidia_smi] : Traceback (most recent call last):
  File "/usr/libexec/netdata/python.d/python_modules/bases/FrameworkServices/SimpleService.py", line 201, in run
    updated = self.update(interval=since)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/libexec/netdata/python.d/python_modules/bases/FrameworkServices/SimpleService.py", line 227, in update
    data = self.get_data()
           ^^^^^^^^^^^^^^^
  File "/usr/libexec/netdata/python.d/nvidia_smi.chart.py", line 556, in get_data
    gpu_data = gpu.data()
               ^^^^^^^^^^
  File "/usr/libexec/netdata/python.d/nvidia_smi.chart.py", line 489, in data
    data['rx_util_percent'] = str(int(int(self.rx_util())*100/self.pci_bw_max()))
                                      ^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'N/A'
```

**Comment by koebi001** [comment]
to fix this in file `collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py` change line `484` form:
```
        pci_bw_max = self.pci_bw_max()
```
to
```
       if not pci_bw_max or not self.rx_util().isdigit() or not self.tx_util().isdigit():
```

**Comment by ilyam8** [comment]
Thanks for reporting, @koebi001. Fixed in #15231.
